src/commands/build.ts

```diff
@@ -44,11 +59,6 @@ export class BuildCommand implements Command {
           'additional `<link rel="prefetch">` tags into ' +
           'entrypoint and `<link rel="import">` tags into fragments and shell.'
     },
-    {
-      name: 'html.collapseWhitespace',
```

Comment:
I'm not against opening explicit configuration options back up in `polymer.json`, but I have a hard time picturing anyone setting specific optimization JSON options objects via CLI flags every time they want to build. At that point that user really should be saving their build configuration in a polymer.json. I'd rather we encourage them to do that and keep these CLI flags as simple as possible.

Again, I don't think we should spend time at the moment worrying about how these CLI flags will match the polymer.json interface, when that interface doesn't even exist yet. I'd much rather discuss that when we do add the polymer.json interface, either in a design proposal or the PR itself. None of this is going out to users until then, so there is no difference between discussing in this PR vs. that PR, except that we'll have both interfaces concretely in front of us and will be able to reason about them much more easily.